### PR TITLE
mock footer done

### PR DESCRIPTION
**Pull request by ZahraaTayyar**
- created start of a footer
- added links to cyf homepage, our github repo and cyf instagram (just to see how it looks, none of this is final)
- will add icons

### CODE REVIEW

**Review by Bahare09 — APPROVED** [review]

File: client/src/pages/Footer.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+				</Link>
+			</div>
+			<div className="text-container">
+				<p>
+					This website was created in association with CodeYourFuture
+					<span>
+						<img
+							src="https://codeyourfuture.io/wp-content/uploads/2019/03/cyf_brand.png"
+							alt="cyf logo"
+						></img>
```
Comment (LexiisYing) [inline client/src/pages/Footer.js:33]: image tags should be self closing.

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -24,9 +28,12 @@ export function Home() {
 
 	return (
 		<main role="main">
+
```
Comment (LexiisYing) [inline client/src/pages/Home.js:31]: it looks like there's some issue with the indentation here, can we try to sort it out plz.

File: server/package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC"
+}
```
Comment (LexiisYing) [inline server/package.json:12]: how come this is being committed? i thought we only needed it in the root directory?

**Review by LexiisYing — COMMENTED** [review]
hey, i have reviewed it and there are things that need to be changed before we can merge it plz.

File: server/package.json (hunk 3 above)
Comment (ZahraaTayyar) [reply server/package.json:12]: I don't really know, i'm not very familiar with what that is

**Review by LexiisYing — APPROVED** [review]

File: server/package.json (hunk 3 above)
Comment (Bahare09) [reply server/package.json:12]: @ZahraaTayyar  it is there because you installed npm on your server.

**Review by LexiisYing — APPROVED** [review]

**Pull request merged**